rstest/tests/fixture/mod.rs

```diff
@@ -378,6 +378,21 @@ mod should {
             run_test("errors_once.rs")
         }
 
+        #[fixture]
+        #[once]
+        fn deny_docs_rs() -> (Output, String) {
+            run_test("deny_docs.rs")
+        }
+
+        #[rstest]
+        fn deny_docs(deny_docs_rs: &(Output, String)) {
```

Comment:
My taste is that introduce a fixture here to use it only once is a little bit overdoing... bu it's fine :smile:

Reply:
Oh, right, I can just inline that. Wait a bit. I was just looking at other examples :sweat_smile:

Reply:
Okay, should look a bit better now :)